### PR TITLE
Prevent "FileSystem closed" error for Hive connector

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -351,7 +351,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hadoop</groupId>
                 <artifactId>hadoop-apache2</artifactId>
-                <version>2.7.3-1-t</version>
```
Comment (akshatnair) [inline pom.xml:354]: Check what was the version used for 167t-0.7 and 0.6 to make sure we use open source version and not our version

Comment (sanjay990) [reply pom.xml:354]: It was changed from 0.10(167t-0.6) to 2.7.3-1-t(in 167t-0.7).  For this release we are using open source version.

**Comment by akshatnair** [comment]
Okay lgtm, just make sure that build is failing due to a valid reason and not a real issue.

**Comment by sanjay990** [comment]
As per my analysis, it looks like an intermittent issue. It initially passed on `release-0.167-t`, and its failing now ( we haven't modified the brach ). I tested locally and the tests pass.

**Pull request merged**